src/components/Balance/Balance.jsx

```diff
@@ -0,0 +1,59 @@
+import * as React from 'react';
+import { useSelector } from 'react-redux';
+import Box from '@mui/material/Box';
+import Container from '@mui/material/Container';
+// import css from '../../utils/css-variables.css';
+
+import { formatMoney } from '../../utils/formatMoney';
+import { selectBalance } from '../../redux/store';
+
+export default function Balance() {
+  const balance = useSelector(selectBalance);
+  return (
+    <Container
+      sx={{
+        '@media (min-width: 780px)': {
+          width: '360px',
+        },
+        '@media (min-width: 1200px)': {
+          width: '395px',
+        },
+        width: '280px',
+        backgroundColor: 'white',
+        borderRadius: 30,
+        height: 'auto',
+        textTransform: 'uppercase',
+        marginBottom: 32,
+        fontSize: 12,
```

Comment:
font-size nie potrzebny
za to przydałby się padding, żeby ustawić elementy w środku:
padding: '8px 32px 11px'
i od tabletu '8px 40px 11px'